int around() : call(int foo2(int)) {
	printf("aspect, call proceed \n");
	proceed();
	
	printf("aspect, throw 3\n");
	throw(3);
	printf("end of aspect\n");
}

catch(int e): try(call(int foo(int))) {
	printf("catch exception = %d\n", e);
}

